scripts/get-sampleapp-url.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+#########################################################################################################
+# Script to retrieve the websphere sample app url from the openshift routes in the sample app namespace
+#########################################################################################################
+
+import json
+import logging
+import os
+import subprocess
+import sys
+import time
+
+input = sys.stdin.read()
+input_json = json.loads(input)
+
+# json response format
+json_response = {"response": ""}
+
+# debugging file init
+DEBUGFILE = "/tmp/getsampleappurl.log"
```

Comment:
https://docs.python.org/3/library/tempfile.html
Can we use temp file here and print the path to console